src/views/Login/index.js

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
+import { Link } from 'react-router-dom'
+import './login.css';
 
-const Login = () => <h1>Edustripe Login page</h1>;
+import edustripeLogo from '../../assets/images/Edustripelogo.png';
+
+const Login = () => {
+  return (
+    <div className="login-page">
+      <div className="logo">
+        <img alt="editripe logo" src={edustripeLogo} />
+      </div>
+      <p className="welcome-back">Welcome back</p>
+      <div className="form-div">
+        <form className="login-form">
+          <div className="input-div">
+            <label htmlFor="email">Email:
+            </label>
+            <input className="input" type="email" required />
+          </div>
+          <div className="input-div">
+            <label htmlFor="password">Password:
+            </label>
+            <input  className="input" type="password" required />
+          </div>
+          <input className="submit" type="submit" value="Sign in" />
+        </form>
+      </div>
+      <div>
+        <p className="txt">Don’t have an account? Click here to <Link to="/parent-signup">sign up</Link></p>
+      </div>
+    </div>
+  );
+}
```

Comment:
This is nice work @obayomi96. But in the future, I'll suggest we create reusable input elements so as to enable us reuse logic across the app.

Reply:
Thanks, I know that normally, but I also know very few people will use it, I had to put it down for now till our app gets more robust.

Reply:
I can implement it along with the new PR coming up for our landing page though.

Reply:
All right man. Makes sense.